targets/targets.json

```diff
@@ -4698,6 +4698,18 @@
         "features": ["BLE"],
         "release_versions": ["2", "5"]
     },
+    "ARM_MUSCA_A1": {
+        "inherits": ["Target"],
+        "core": "Cortex-M33-NS",
+        "device_has": ["INTERRUPTIN", "LPTICKER", "SERIAL", "SLEEP", "USTICKER"],
+        "macros": ["MBED_TZ_DEFAULT_ACCESS=1", "__STARTUP_CLEAR_BSS", "MBED_FAULT_HANDLER_DISABLED", "CMSIS_NVIC_VIRTUAL", "LPTICKER_DELAY_TICKS=1"],
+        "extra_labels": ["ARM_SSG", "MUSCA_A1"],
+        "supported_toolchains": ["ARMC6", "GCC_ARM"],
```

Comment:
If release_versions : 2 and 5, this should have IAR support as well (@ashok-rao correct for m33 as well?)

Is version 2 correct in this case?

Reply:
@gaborkertesz : Could you please remove support for Mbed 2. Thanks.

Reply:
Probably version 2 is coming from not correct understanding and legacy reasons, will be removed.

Reply:
Fix commit has been submitted.